docs/octopus-cloud/index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Octopus Cloud
+position: 10
+description: How to work with Octopus Cloud.
+---
+**Octopus Cloud** is the hosted version of Octopus Deploy and it has been publicly available since February 2018. We designed Octopus Cloud and self-hosted Octopus to provide the same functionality; however, there are some minor differences, for instance, with Octopus Cloud, we're [responsible](/docs/administration/security/index.md#responsibility) for taking backups, automatically upgrading the service, and maintaining and monitoring the underlying systems, and for security reasons some of the configuration and diagnostic functionality has been limited. <!-- do we have specifics? -->
```

Comment:
I'd possibly defer this to @pawelpabich's judgement about how much he is willing to share here

Reply:
This looks good to me. 👍